docs/static-analysis.md

```diff
@@ -259,18 +255,10 @@ This means they automatically run on every open file in the editor. Any code iss
 editor and you can also see the analysis status of the whole file from by hovering over the icon on the top-right of the
 editor. You may also be able to see some code inspection status on the line itself (e.g. wriggly red lines).
 
-`FindBugs` is also an inspection tool but it does not run automatically. You can run it by going to `Analyze → FindBugs`
```

Comment:
Instructions for SpotBugs also?

Reply:
There aren't special plugin to Intellij - the best way is to use `gradle lint`.

Reply:
I don't mean to intrude, but there is definitely a way to use SpotBugs as it is in the current state (with IDE integration). The problem is whether to integrate manually with `external tool` or wait for the IntelliJ plugin to be ready.

`gradle lint` worsens the developer experience by a lot.

Reply:
@LiHaoTan I think it's OK to wait for the IntelliJ plugin to be ready as we only have one rule in findBugs currently.

Reply:
I've added Intellij external tool guide. But is impossible to include it in intellij automatic setup. We have to wait for plugin